nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -72,6 +72,12 @@ BtNavigator::BtNavigator()
   declare_parameter("global_frame", std::string("map"));
   declare_parameter("robot_base_frame", std::string("base_link"));
   declare_parameter("odom_topic", std::string("odom"));
+  declare_parameter("pose_enable_groot_monitoring", true);
```

Comment:
If we have these here, we should outright remove the parameter gets / declares in the Bt action node 
- https://github.com/ros-planning/navigation2/blob/8947c9fdca621624fddc49ff40af9d3f29693e81/nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp#L62-L70
- https://github.com/ros-planning/navigation2/blob/8947c9fdca621624fddc49ff40af9d3f29693e81/nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp#L101-L103

And then we only have the API to set the ports from the application